samcli/commands/sync/command.py

```diff
@@ -77,20 +70,8 @@
 Enter Y to proceed with the command, or enter N to cancel:
 """
 
-SYNC_CONFIRMATION_TEXT_WITH_BETA = """
-This feature is currently in beta. Visit the docs page to learn more about the AWS Beta terms https://aws.amazon.com/service-terms/.
-
-The SAM CLI will use the AWS Lambda, Amazon API Gateway, and AWS StepFunctions APIs to upload your code without 
-performing a CloudFormation deployment. This will cause drift in your CloudFormation stack. 
-**The sync command should only be used against a development stack**.
-
-Confirm that you are synchronizing a development stack and want to turn on beta features.
-
-Enter Y to proceed with the command, or enter N to cancel:
-"""
```

Comment:
We still want this confirmation in the beginning of `sam sync`. So we only need to remove first line here, rest should stay there and we should get a confirmation from customer every time they run `sam sync`.

Reply:
We already have one name `SYNC_CONFIRMATION_TEXT` above, just remove this beta one.

Reply:
Got it, I thought we had one text and we display it all the time. Thanks!